cmd/root.go

```diff
@@ -4,14 +4,15 @@ import (
 	"io"
 	"os"
 
-	"github.com/giantswarm/kubectl-gs/cmd/template"
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
 	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/kubectl-gs/cmd/template"
 )
 
 const (
-	name        = "kubectl-gs"
+	name        = "kubectl" + string(0xA0) + "gs"
```

Comment:
Maybe add the explanation for using the non breakable space also here in the code as a comment?